### PR TITLE
support KL label smooth

### PR DESCRIPTION
**Pull request by QingshuChen**
### PR types
Others

### PR changes
OPs

### Describe
support KL label smooth

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/xpu/test_label_smooth_op_xpu.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    def setUp(self):
+        self.config()
+        smoothed_label = (1 - self.epsilon
+                          ) * self.label + self.epsilon / self.label_dim
+        self.inputs = {'X': self.label}
+        self.attrs = {'epsilon': self.epsilon}
+        self.outputs = {'Out': smoothed_label}
+
+    def test_check_output(self):
```
Comment (tangzhiyi11) [inline python/paddle/fluid/tests/unittests/xpu/test_label_smooth_op_xpu.py:45]: 单测需要用 get_xpu_version 来区分kl1和kl2么？

Comment (QingshuChen) [reply python/paddle/fluid/tests/unittests/xpu/test_label_smooth_op_xpu.py:45]: 对于能切换到CPU的，可以不用加

Comment (tangzhiyi11) [reply python/paddle/fluid/tests/unittests/xpu/test_label_smooth_op_xpu.py:45]: ok

**Review by qili93 — APPROVED** [review]
LGTM for atol

**Pull request merged**